rpctest/rpcharness_test.go

```diff
@@ -2123,13 +2123,6 @@ func testGetSetTicketMaxPrice(r *Harness, t *testing.T) {
 	if err != nil {
 		t.Fatal("SetTicketMaxPrice failed:", err)
 	}
-
```

Comment:
I pushed right before you committed the change in dcrd:

https://github.com/decred/dcrd/commit/10adc627c8e04c8888eb9af9fa90b5d401013dc0

So it is unfortunately already removed in dcrd and thus is necessary to remove here too.